module/x/gravity/types/ethereum.go

```diff
@@ -68,6 +68,10 @@ func (e *ERC20Token) GravityCoin() sdk.Coin {
 	return e.Coin
 }
 
+func NewERC20TokenFromCoin(coin sdk.Coin) ERC20Token {
```

Comment:
Is this function really necessary?

Reply:
Not as currently written. I was thinking of adding some validation to make sure its an ERC20, but that can be skipped or put elsewhere